tests/e2e/lib.sh

```diff
@@ -146,7 +146,6 @@ export_test_environment() {
     ci_export ROX_ADMINISTRATION_EVENTS "${ROX_ADMINISTRATION_EVENTS:-true}"
     ci_export ROX_POLICY_CRITERIA_MODAL "${ROX_POLICY_CRITERIA_MODAL:-true}"
     ci_export ROX_TELEMETRY_STORAGE_KEY_V1 "DISABLED"
-    ci_export ROX_SCANNER_V4_SUPPORT "${ROX_SCANNER_V4_SUPPORT:-true}"
```

Comment:
This is what I have been referring to. I was about to explain why I think we should keep this one. But then I realized that the corresponding feature flag `ScannerV4Support` is an **unchangeable** feature flag nowadays, hence it shouldn't matter what `ROX_SCANNER_V4_SUPPORT` is set to in the environment. (Correct, @dcaravel?)

Hence, should be fine to remove as a cleanup.

Reply:
Oh, I see. Yeah, I couldn't find a reference to this variable in any of the shell scripts. Do you know how that variable is propagated? It looked like `ROX_SCANNER_V4` was the variable that controlled all the scanner v4 enabling.

Reply:
> This is what I have been referring to. I was about to explain why I think we should keep this one. But then I realized that the corresponding feature flag ScannerV4Support is an unchangeable feature flag nowadays, hence it shouldn't matter what ROX_SCANNER_V4_SUPPORT is set to in the environment. (Correct, @dcaravel?)

Agreed

Reply:
Suppose to further qualify - unchangeable feature flags are changeable in non-release builds - in this case there are no plans to change the default for `ROX_SCANNER_V4_SUPPORT` so OK with the removal, or if prefer to be more explicit OK keeping it as well.

Reply:
It's still not clear to me what `ROX_SCANNER_V4_SUPPORT` does so I feel compelled to keep it but default it to false. If anyone feels differently (i.e., changing it back to true or removing it entirely), please feel free to submit that change 🙂

Reply:
I think since there are no plans to change the defaults for ROX_SCANNER_V4_SUPPORT, removal would be preferable.

I would advice against setting it to false, because in that case non-release builds and release-builds would -- IIUC -- actually test slightly different code paths in several places.

I see no good reason for setting to false. In a nutshell: `ROX_SCANNER_V4_SUPPORT` can be described as the build-time flag for Scanner V4 *support* in different components (Sensor, Central, Helm Charts, etc.). When enabled (the "semi-unchangeable" default) these components know in principle how to communicate with Scanner V4 or how to install it.

This is clearly different from *enabling* or *actively using* Scanner V4, which is controlled by `ROX_SCANNER_V4`. For example, when this is enabled, Central and Sensor expect that there is a Scanner V4 deployment and tries talking to it.

Reply:
Understood. Thank you for the detailed explanation 🙂